dashboard/lib/queries/school_info.rb

```diff
@@ -2,4 +2,34 @@ class Queries::SchoolInfo
   def self.last_complete(user)
     Queries::UserSchoolInfo.last_complete(user)&.school_info
   end
+
+  def self.current_school(user)
+    user_school_info = Queries::UserSchoolInfo.last_complete(user)
```

Comment:
My brain is foggy on where we landed re: saving new rows in the school_info table and having 'last_complete' always be accurate. Because we're collecting additional data (and not just country), that's no longer a concern right?

Reply:
exactly! in the refactored components that use `SchoolDataInputs` it's not possible to send "incomplete" school info. it's either an nces school with a schoolId or a school with a country and school name, so all new entries into `school_infos` table are considered complete. this is probably another spot to clean up! can just get the last entry for the user

![Screenshot 2024-11-05 at 8 52 05 AM](https://github.com/user-attachments/assets/e92043d0-797f-4d28-b629-a2b96a0b0b20)